Install `conda-lock` on startup for Dev Containers (#35)
konstantinjdobler/nlp-research-template: README.md

<p>

> The preferred method is to install conda-lock into your `mamba` / `conda` `base` environment using `mamba install -c conda-forge -n base conda-lock`. Then, you can access conda-lock via the automatic subcommand discovery (e.g. `mamba lock --version`). Otherwise, visit the [conda-lock repo](https://github.com/conda/conda-lock). Conda-lock is installed in the devcontainer during start. For basic usage, have a look at the commands below:

Reviewer: ```suggestion
The preferred method is to install `conda-lock` using `pipx install conda-lock`. For other options, visit the [conda-lock repo](https://github.com/conda/conda-lock). For basic usage, have a look at the commands below:
```
